carbonmark/.generated/carbonmark-api.schema.ts

```diff
@@ -12,7 +12,7 @@ export default {
       "name": "MIT",
       "url": "https://github.com/KlimaDAO/klimadao/blob/main/LICENSE"
     },
-    "version": "2.0.0-6"
+    "version": "2.0.0-8"
```

Comment:
how were you able to generate this if v2.0.0-8 does not have a release yet?

This is why the build fails i assume

Reply:
1. Run the local dev server: `npm run dev-carbonmark-api`
2. run `cd carbonmark && NEXT_PUBLIC_CARBONMARK_API_URL=http://localhost:3003 npm run generate:types` 
3. Set the `NEXT_PUBLIC_USE_PREVIEW_CARBONMARK_API` in vercel to this branch (the first build when doing this will always fail because the preview deployment hasn't finished until after the first pass)